packages/env/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@latitude-data/typescript-config/base.json",
+  "compilerOptions": {
+    "module": "ESNext",
+    "moduleResolution": "node",
+    "rootDir": "./src"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules"]
+}
```

Comment:
it's so bad that we need so much configuration for a pkg that just exports a single function that took 3 minutes to write honestly

Reply:
🤷‍♂️